packages/emitter-framework/src/python/components/dictionary-expression.tsx

```diff
@@ -0,0 +1,17 @@
+export interface DictionaryExpressionModel {
+  /** The value type of the Dictionary. */
+  // COMMENT: Should this be elementType (same in ArrayExpressionModel)?
```

Comment:
no strong opinion, but we have it as `elementType` in the typescript components. Might be good to standardize, open to a different name convention though.

Reply:
I'm fine with elementType